src/more-infos/more-info-climate.html

```diff
@@ -50,22 +50,38 @@
         width: 100%;
       }
 
-      .heat {
+      #temp-slider,
```

Comment:
Instead of IDs you can just use the tag name `paper-slider` as we're in shadow DOM